src/editor/MessageToggle.vue

```diff
@@ -0,0 +1,66 @@
+<script setup lang="ts">
```

Comment:
Please format the code `pnpm run format`

Reply:
Done
But there are two problems with prettier
* prettier is not in the package.json as devDependencies
* `pnpm run format` will format all files. It is better to format the changed files only by running `prettier path/to/changed/file --write`